src/patchelf.cc

```diff
@@ -1585,7 +1591,7 @@ template<class ElfFile>
 static void patchElf2(ElfFile && elfFile, const FileContents & fileContents, std::string fileName)
 {
     if (printInterpreter)
-        printf("%s\n", elfFile.getInterpreter().c_str());
+        printf("INTERPRETER: %s\n", elfFile.getInterpreter().c_str());
```

Comment:
Sorry. We cannot merge this as this would break our use in nixpkgs, which expects no prefix in many places.